Index.html

```diff
@@ -0,0 +1,221 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Type coercion operations</title>
+</head>
+<body>
+    <p>Type coertion operations</p>
+    <script>
```

Comment:
Maybe, it's better to pull all js code in a separate file? `main.js` as example.